e2e/suites/provider/cases/common/common.go

```diff
@@ -512,7 +512,7 @@ func DockerJSONConfig(f *framework.Framework) (string, func(*framework.TestCase)
 	return "[common] should sync docker configurated json secrets with template simple", func(tc *framework.TestCase) {
 		cloudSecretName := fmt.Sprintf("%s-%s", f.Namespace.Name, dockerConfigExampleName)
 		cloudRemoteRefKey := f.MakeRemoteRefKey(cloudSecretName)
-		dockerconfig := `{"auths":{"https://index.docker.io/v1/": {"auth": "c3R...zE2"}}}`
+		dockerconfig := `{"auths":{"https://index.docker.io/v1/":{"auth":"c3R...zE2"}}}`
```

Comment:
DSV only supports JSON values as secrets. Therefore, the secret entries supplied by the E2E framework are unmarshalled into a map (see https://github.com/external-secrets/external-secrets/pull/2415/files#diff-e2aa126d5482f9f14b146b6bf57efbd593114f35071fc42a0959b557ee47bc8eR36). This has the effect that the secret synced from the DSV has all whitespace trimmed.

I believe the tests of the other providers should work with the trimmed whitespace. If not, I can revert this but then need to remove `common.DockerJSONConfig` from the E2E tests.